src/include/statistics/backend_stats_context_new.h

```diff
@@ -0,0 +1,115 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// backend_stats_context.h
+//
+// Identification: src/statistics/backend_stats_context.h
```

Comment:
Eh this is again confusing. I think we only need to keep one of this class, either the old one or the new one. Can you explain what's going on here?